src/Servers/Connections.Abstractions/src/HttpEndPoint.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+
+namespace Microsoft.AspNetCore.Connections
+{
+    public class HttpEndPoint : EndPoint
```

Comment:
I wonder if this should be in abstractions or in the https.connections.common assembly.

Reply:
I followed the pattern you used for FileHandleEndPoint down to the namespace and lack of virtual properties.